embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/SystemInfo.kt

```diff
@@ -7,7 +7,7 @@ import io.opentelemetry.semconv.incubating.OsIncubatingAttributes
  * Information about the the device or OS that can be retrieved without disk or platform API access
  */
 internal data class SystemInfo(
-    val osName: String = "android",
+    val osName: String = "Android OS",
```

Comment:
I want to rename this value to `android` but will need to coordinate with the backend folks first.

Reply:
I'm going to rename this android now since the backend supports this too